storage/innobase/include/data0type.h

```diff
@@ -315,6 +320,16 @@ ulint dtype_get_at_most_n_mbchars(
  @return true if string type */
 bool dtype_is_string_type(
     ulint mtype); /*!< in: InnoDB main data type code: DATA_CHAR, ... */
+
+#ifdef XTRABACKUP
+/** Checks if a data main type is a real temporal type.
+ @return true if real temporal type */
+bool dtype_is_real_temporal_type(
+    dd::enum_column_types
+        type); /*!< in: InnoDB main data type code: TINY, SHORT, ... */
```

Comment:
remove the in:.. and make param[in]

Reply:
I copied the format from the previous function declaration. I'll remove it.

Reply:
Done